content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1466,10 +1466,24 @@ This annotation records a comma-separated list of
 managed by [Node Feature Discovery](https://kubernetes-sigs.github.io/node-feature-discovery/) (NFD).
 NFD uses this for an internal mechanism. You should not edit this annotation yourself.
 
+### nfd.node.kubernetes.io/node-name
+
+Type: Annotation
+
+Example: `nfd.node.kubernetes.io/node-name: node-1`
+
+Used on: Nodes
+
+It specifies which node the NodeFeature object is targeting.
+Creators of NodeFeature objects must set this label and 
+consumers of the objects are supposed to use the label for 
```

Comment:
Is this an annotation or a label?
An annotation cannot be used to filter objects.

Reply:
If it is a label, you will need to change line 1471, correct?
On line 1478, you are also referring this as an annotation too.

Reply:
> If it is a label, you will need to change line 1471, correct? On line 1478, you are also referring this as an annotation too.

Done the changes